### PR TITLE
[keras/mixed_precision] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

File: keras/mixed_precision/loss_scale_optimizer.py [hunk 1]
```diff
@@ -406,14 +406,14 @@ class BaseLossScaleOptimizer(metaclass=LossScaleOptimizerMetaclass):
...
         `tf.keras.optimizers.experimental.Optimizer` instance to wrap.
-      dynamic: Bool indicating whether dynamic loss scaling is used. Defaults to
-        True. If True, the loss scale will be dynamically updated over time
-        using an algorithm that keeps the loss scale at approximately its
-        optimal value.  If False, a single fixed loss scale is used and
-        `initial_scale` must be specified, which is used as the loss scale.
+      dynamic: Bool indicating whether dynamic loss scaling is used.  If True,
+        the loss scale will be dynamically updated over time using an algorithm
+        that keeps the loss scale at approximately its optimal value. If False,
+        a single fixed loss scale is used and  `initial_scale` must be
```
Comment (fchollet) [inline keras/mixed_precision/loss_scale_optimizer.py:412]: Remove extra space

File: keras/mixed_precision/loss_scale_optimizer.py [hunk 2]
```diff
@@ -406,14 +406,14 @@ class BaseLossScaleOptimizer(metaclass=LossScaleOptimizerMetaclass):
     Args:
       inner_optimizer: The `tf.keras.optimizers.Optimizer` or
         `tf.keras.optimizers.experimental.Optimizer` instance to wrap.
-      dynamic: Bool indicating whether dynamic loss scaling is used. Defaults to
-        True. If True, the loss scale will be dynamically updated over time
-        using an algorithm that keeps the loss scale at approximately its
-        optimal value.  If False, a single fixed loss scale is used and
-        `initial_scale` must be specified, which is used as the loss scale.
+      dynamic: Bool indicating whether dynamic loss scaling is used.  If True,
```
Comment (fchollet) [inline keras/mixed_precision/loss_scale_optimizer.py:409]: Remove extra space, use backticks for `True`

**Review by haifeng-jin — COMMENTED** [review]
LGTM. No more comments from me.

File: keras/mixed_precision/loss_scale_optimizer.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/mixed_precision/loss_scale_optimizer.py:412]: Done

File: keras/mixed_precision/loss_scale_optimizer.py (hunk 2 above)
Comment (SamuelMarks) [reply keras/mixed_precision/loss_scale_optimizer.py:409]: Fixed

**Review by haifeng-jin — APPROVED** [review]

**Pull request merged**